src/common/_modules/global/modules/dns/outputs.tf

```diff
@@ -36,3 +36,7 @@ output "private_dns_zones" {
     servicebus = azurerm_private_dns_zone.privatelink_servicebus
   }
 }
+
+output "external_domain" {
+  value = var.external_domain
```

Comment:
no way to get it from resource's output?

Reply:
It is used in several resources, I can't find a resource where we can get it straight away

This was put in place when the dns module was used in the core workspace.
After it has been moved to common, I'm torn whether we should remove the output and use a single variable in the common workspace. 

Using the output surely creates a stronger bond between the actual external domain and the resources that make use of it
What do you think?